tensorflow/contrib/tensorrt/convert/convert_nodes.h

```diff
@@ -32,12 +32,13 @@ limitations under the License.
 
 namespace tensorflow {
 namespace tensorrt {
+static const string kInputPHName = "InputPH_";
```

Comment:
Ok. Will do, but I added them as strings since I saw examples of static const string being used elsewhere such as in common_runtime, grappler and framework.